File: data_structures/q10/solution.cpp
#include<iostream>
#include<vector>
using namespace std;

void duplicateElements(vector<int> arr){
    sort(arr.begin(),arr.end());
    bool found = false;

    for(int i = 0; i < arr.size(); i++){
        if(arr[i] == arr[i-1]){
            cout << arr[i] << " ";
            found = true;

            while(i < arr.size() && arr[i] == arr[i-1]){
                i++;
            }
        }
    }
    if(!found){
        cout << "None";
    }
}

int main(){
    vector<int> arr = {1,9,3,4,5,6,4,2,2,1};
    cout << "The repeating elements are: ";
    duplicateElements(arr);

    return 0;
}

File: data_structures/q10/solution.py
def duplicateElements(arr):
    arr.sort()
    found = False

    for i in range(len(arr)):
        if(arr[i] == arr[i-1]):
            print(arr[i],end =" ")
            found = True

            while(i < len(arr) and arr[i] == arr[i-1]):
                i+=1

    if not found:
        print("None")

arr = [1,3,2,1,2,4]
print("The repeating elements are: ",end = " ")
duplicateElements(arr)